Change golang version requirement

Use 1.11 instead of 1.13 as 1.13 is not packaged for scibian9.

===================================================================
--- a/mconfig
+++ b/mconfig
@@ -20,7 +20,7 @@
 hstranlib=
 hstobjcopy=
 hstgo=
-hstgo_version="1.13"
+hstgo_version="1.11"
 hstgo_opts="go"
 
 tgtcc=
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/sylabs/singularity
 
-go 1.13
+go 1.11
 
 require (
 	github.com/Netflix/go-expect v0.0.0-20190729225929-0e00d9168667
--- a/internal/pkg/util/goversion/goversion.go
+++ b/internal/pkg/util/goversion/goversion.go
@@ -3,11 +3,11 @@
 // LICENSE.md file distributed with the sources of this project regarding your
 // rights to use or distribute this software.
 
-// +build go1.13
+// +build go1.11
 
 package goversion
 
-// __BUILD_REQUIRES_GO_VERSION_1_13_OR_LATER__ provides a human-readable
+// __BUILD_REQUIRES_GO_VERSION_1_11_OR_LATER__ provides a human-readable
 // error message when building this package with an unsupported version
 // of the Go compiler.
 //
@@ -15,4 +15,4 @@
 // version specified in the build tag above.
 //
 // nolint:golint
-const __BUILD_REQUIRES_GO_VERSION_1_13_OR_LATER__ = uint8(0)
+const __BUILD_REQUIRES_GO_VERSION_1_11_OR_LATER__ = uint8(0)
--- a/internal/pkg/util/goversion/version_check.go
+++ b/internal/pkg/util/goversion/version_check.go
@@ -10,7 +10,7 @@
 // sufficient to trigger a build failure like:
 //
 //     ...
-//     ../internal/pkg/util/goversion/version_check.go:19:9: undefined: __BUILD_REQUIRES_GO_VERSION_1_13_OR_LATER__
+//     ../internal/pkg/util/goversion/version_check.go:19:9: undefined: __BUILD_REQUIRES_GO_VERSION_1_11_OR_LATER__
 //
 //
 // This is based on the technique presented at
@@ -19,4 +19,4 @@
 
 // keep the variable here in sync with the mininum required version
 // specified in goversion.go
-var _ = __BUILD_REQUIRES_GO_VERSION_1_13_OR_LATER__
+var _ = __BUILD_REQUIRES_GO_VERSION_1_11_OR_LATER__
